container-engine-lib/lib/backend_impls/kubernetes/kubernetes_kurtosis_backend/user_services_functions/persistent_directories.go

```diff
@@ -136,17 +123,6 @@ func preparePersistentDirectoriesResources(
 				logrus.Warnf("You'll need to clean up volume claim '%v' manually!", volumeClaimNameStr)
 			}
 		}
-		for volumeNameStr := range volumesCreated {
-			// Background context so we still run this even if the input context was cancelled
-			if err := kubernetesManager.RemovePersistentVolumeClaim(context.Background(), namespace, volumeNameStr); err != nil {
```

Comment:
it was wrong right? because it was deleting a PVC instead of a PV, well it doesn't matter because now it is not necessary

Reply:
Yes, it was wrong!